app/code/Magento/Theme/view/frontend/requirejs-config.js

```diff
@@ -34,7 +34,8 @@ var config = {
     deps: [
         'mage/common',
         'mage/dataPost',
-        'mage/bootstrap'
+        'mage/bootstrap',
+        'Magento_Ui/js/core/app'
```

Comment:
I just looked one more time - this part is related to the Magento_Ui module.
Could you please move it there?

Reply:
You mean here?

https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/view/base/requirejs-config.js

Reply:
no, pls create a new file https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/view/frontend/requirejs-config.js with only this small piece of code